src/include/migraphx/op/rnn_last_cell_output.hpp

```diff
@@ -1,33 +1,16 @@
 #ifndef MIGRAPHX_GUARD_OPERATORS_RNN_LAST_CELL_OUTPUT_HPP
 #define MIGRAPHX_GUARD_OPERATORS_RNN_LAST_CELL_OUTPUT_HPP
 
-#include <array>
-#include <migraphx/operation.hpp>
-#include <migraphx/check_shapes.hpp>
-#include <migraphx/stringutils.hpp>
-#include <migraphx/streamutils.hpp>
-#include <migraphx/literal.hpp>
-#include <migraphx/shape_for_each.hpp>
-#include <migraphx/config.hpp>
-#include <cmath>
-#include <utility>
+#include <migraphx/op/rnn_last_output.hpp>
 
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace op {
 
-struct lstm_last_cell_output
+struct lstm_last_cell_output : rnn_last_output<lstm_last_cell_output>
```

Comment:
We should just add `rnn_last_hs_output` and `rnn_last_cell_output` with the understanding that `rnn_last_cell_output` is only used for lstms.

Reply:
you mean rename the file name from rnn_last_cell_output.hpp to lstm_last_cell_output here?

Reply:
I actually meant rename the operator to `rnn_last_cell_output` so it matches the filename.

Reply:
fixed